scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -1021,7 +1021,7 @@ class Router(formatOps: FormatOps) {
               Split(Space, 0),
               Split(Newline, 1).withPolicy(policy)
             )
-          case t @ SelfAnnotation(top) =>
+          case t @ WithChain(top) =>
```

Comment:
I don't love this name but I wanted something more general than `SelfAnnotation`

Reply:
This name is good :+1: